oracle/README.md

```diff
@@ -207,6 +205,11 @@ If you are not using JDBC, verify that the Datadog Agent is able to connect to y
 sqlplus <USER>/<PASSWORD>@(DESCRIPTION=(ADDRESS_LIST=(ADDRESS=(PROTOCOL=TCPS)(HOST=<HOST>)(PORT=<PORT>))(SERVICE_NAME=<SERVICE_NAME>)))
 ```
 
+*Note*: When using the [Oracle Instant Client][16] connection, move three files to the `network/admin` directory of the client libraries used by your application:
```

Comment:
I'm a bit confused about this `network/admin` directory. Would this be the same regardless of OS? Did we need to do this back when we supported the instant client with the old library?

Reply:
Nope we didn't need to do this with the old client but we did need to use the `TNS_ADMIN` environment variable, however it was never explicitly mentioned. We can use the same variable but in the case it's not set, I just mentioned where python-oracle looks by default.
> Since the tnsnames.ora and sqlnet.ora files are not in the default location, your application needs to indicate where they are, either with the config_dir parameter to [oracledb.init_oracle_client()](https://python-oracledb.readthedocs.io/en/latest/api_manual/module.html#oracledb.init_oracle_client), or using the TNS_ADMIN environment variable. See [Optional Oracle Net Configuration Files](https://python-oracledb.readthedocs.io/en/latest/user_guide/initialization.html#optnetfiles). (Neither of these settings are needed, and you do not need to edit sqlnet.ora, if you have put all the files in the network/admin directory.)

https://python-oracledb.readthedocs.io/en/latest/user_guide/connection_handling.html#install-the-wallet-and-network-configuration-files

Reply:
```suggestion
When using the [Oracle Instant Client][16] connection, move three files to the `network/admin` directory of the client libraries used by your application:
```